src/annotator/integrations/pdf.js

```diff
@@ -89,14 +89,46 @@ export class PDFIntegration {
     this._destroyed = true;
   }
 
+  /**
+   * Return the URL of the currently loaded PDF document.
+   */
   uri() {
     return this.pdfMetadata.getUri();
   }
 
+  /**
+   * Return the metadata (eg. title) for the currently loaded PDF document.
+   */
   getMetadata() {
     return this.pdfMetadata.getMetadata();
   }
 
+  /**
+   * Resolve serialized `selectors` from an annotation to a range.
+   *
+   * @param {HTMLElement} root
+   * @param {Selector[]} selectors
+   * @return {Promise<Range>}
+   */
+  anchor(root, selectors) {
+    // nb. The `root` argument is not really used by `anchor`. It existed for
+    // consistency between HTML and PDF anchoring and could be removed.
+    return anchor(root, selectors);
+  }
+
+  /**
+   * Generate selectors for the text in `range`.
+   *
+   * @param {HTMLElement} root
+   * @param {Range} range
+   * @return {Promise<Selector[]>}
+   */
+  describe(root, range) {
```

Comment:
For me (newbie), `describe` is a very vague name that doesn't tell me much about the aim of the property.

Reply:
I'm open to changing the term, but I think we'd need to change it everywhere for consistency. Another option is that we could treat it as one of a modest number of domain-specific terms (like "anchor" or "selector") which a newbie will need to look up when they first see it, but will then know what it means the next time they encounter it.